product/views/PhysicalStorage.yaml

```diff
@@ -28,7 +29,6 @@ col_order:
 - name
 - type
 - health_state
-- power_state
```

Comment:
@agrare Does this make sense to you?

I do not know how this works because `PhysicalStorage#power_state` does not exist.

maybe I should move this out of here.

Reply:
Yeah I don't know what they're referencing here, physical storage canisters (groups of physical disks in a physical storage) does have power_state, maybe they thought physical_storages must also?

Reply:
oh cool?
If it is not a valid field, it does not show up in the sql. (probably blows up later on but since I do not have physical servers, it is "working" locally for me)